OWNERS_ALIASES

```diff
@@ -31,39 +30,29 @@ aliases:
   - enxebre
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for docs/book
+  # OWNER_ALIASES for bootstrap/kubeadm
   # -----------------------------------------------------------
 
-  # folks who can review and LGTM any PRs under docs/book
-  cluster-api-book-reviewers:
-  - moshloop
-  - randomvariable
+  cluster-api-bootstrap-provider-kubeadm-maintainers:
+  cluster-api-bootstrap-provider-kubeadm-reviewers:
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for test/infrastructure/docker
+  # OWNER_ALIASES for controlplane/kubeadm
   # -----------------------------------------------------------
 
-  cluster-api-provider-docker-maintainers:
-  - fabriziopandini
+  cluster-api-controlplane-provider-kubeadm-maintainers:
+  cluster-api-controlplane-provider-kubeadm-reviewers:
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for bootstrap/kubeadm
+  # OWNER_ALIASES for cmd/clusterctl
   # -----------------------------------------------------------
 
-  cluster-api-bootstrap-provider-kubeadm-maintainers:
-  - fabriziopandini
-  - SataQiu
-
-  cluster-api-bootstrap-provider-kubeadm-reviewers:
-  - fabriziopandini
+  cluster-api-clusterctl-maintainers:
+  cluster-api-clusterctl-reviewers:
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for cmd/clusterctl
+  # OWNER_ALIASES for test/infrastructure/docker
```

Comment:
I wonder if we should scope this instead with test/ or need other aliases for the other test/* folders. But of course we can always add those when necessary.